test/blocks/ost/mocks/wcs-artifacts-mock.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
this file should be minimised to contain only the minimal set of data and real ids/codes should be replaced with dumb ones.
All non used properties should be removed.